src/OperationResponse.php

```diff
@@ -127,9 +154,16 @@ public function __construct($operationName, $operationsClient, $options = [])
      */
     public function isDone()
     {
-        return (is_null($this->lastProtoResponse) || is_null($this->lastProtoResponse->getDone()))
-            ? false
-            : $this->lastProtoResponse->getDone();
+        if (!$this->hasProtoResponse()) {
+            return false;
+        }
+
+        $status = call_user_func([$this->lastProtoResponse, $this->operationStatusMethod]);
```

Comment:
Same thing here regarding `call_user_func` vs. variable function calls.

Reply:
As mentioned above, here it does matter, because you can't call a property as a method without saving it as a local variable.